platform/authorized_api_guide.md

```diff
@@ -38,22 +73,247 @@ curl 'https://<domain>/auth/token' -H "Content-Type: application/json" --data '{
 }' 
 ```
 
-The example response is: 
+Response example:
 ```
 {"token_type":"Bearer","expires_in":86400,"access_token":"your_access_token"}
 ```
 
-Piwik PRO API tokens use [jwt](https://jwt.io/) format.
+Field `access_token` contains your token which then will be used for all API calls.
```

Comment:
I'm not sure. We have written it is JWT, we have written it is a token. It could be easily googleable. I'm not sure what technical knowledge of end users will be.

Reply:
in layman's terms token is "d1389hf31298fh1" or "18417", but not long elegy of weird symbols and dots. maybe we could show real JWT in example response from oauth?

Reply:
it would not hurt anybody, but it could only help